PLog.cs

```diff
@@ -1,13 +1,45 @@
-﻿using System;
-
-namespace p2pcopy
-{
-	public class PLog
-	{
-		public static void DEBUG(string fmt, params object[] args)
-		{
-			//Console.WriteLine(string.Format(fmt, args));
-		}
-	}
-}
-	
\ No newline at end of file
+﻿using System;
+
+namespace p2pcopy
+{
+    public class PLog
+    {
+        private static bool debug = false;
+        private static bool verbose = false;
+
+        public static void SetDebug()
+        {
+            debug = true;
+        }
+
+        public static void SetVerbose()
+        {
+            verbose = true;
+        }
+
+        public static void Debug(string fmt, params object[] args)
+        {
+            if (debug)
+            {
+                Console.WriteLine (string.Format (fmt, args));
+            }
+        }
+
+        public static void Verbose(string fmt, params object[] args)
+        {
+            if (verbose || debug)
+            {
+                Console.WriteLine(string.Format(fmt, args));
+            }
+        }
+
+        public static void VerboseWrite(string fmt, params object[] args)

```

Comment:
if this is called VerboseWrite, for consistency the others should be called VerboseWriteLine and DebugWriteLine

Reply:
Done!